frontend/talentsearch/src/js/components/applicantProfile/ExperienceAndSkills.tsx

```diff
@@ -0,0 +1,365 @@
+  const intl = useIntl();
+  const compareByDate = (e1: any, e2: any) =>
+    e1.endDate && e2.endDate
+      ? new Date(e2.endDate).getTime() - new Date(e1.endDate).getTime()
+      : 0;
+  const awardExperiences =
+    experiences
+      ?.filter((experience) => isAwardExperience(experience))
+      .map(
+        (award: AwardExperience) =>
+          ({
+            ...award,
+            endDate: award.awardedDate,
+          } as AwardExperience),
+      )
+      .sort(compareByDate) || [];
+  const communityExperiences =
+    experiences
+      ?.filter((experience) => isCommunityExperience(experience))
+      .sort(compareByDate) || [];
+  const educationExperiences =
+    experiences
+      ?.filter((experience) => isEducationExperience(experience))
+      .sort(compareByDate) || [];
+  const personalExperiences =
+    experiences
+      ?.filter((experience) => isPersonalExperience(experience))
+      .sort(compareByDate) || [];
+  const workExperiences =
+    experiences
+      ?.filter((experience) => isWorkExperience(experience))
+      .sort(compareByDate) || [];
+
+  const links = [
+    {
+      href: "/",
+      title: intl.formatMessage({
+        defaultMessage: "Personal",
+        description: "Title for personal experience form button.",
+      }),
+      icon: <LightBulbIcon style={{ width: "1.5rem" }} />,
+    },
+    {
+      href: "/",
+      title: intl.formatMessage({
+        defaultMessage: "Community",
+        description: "Title for community experience form button.",
+      }),
+      icon: <UserGroupIcon style={{ width: "1.5rem" }} />,
+    },
+    {
+      href: "/",
+      title: intl.formatMessage({
+        defaultMessage: "Work",
+        description: "Title for work experience form button.",
+      }),
+      icon: <BriefcaseIcon style={{ width: "1.5rem" }} />,
+    },
+    {
+      href: "/",
+      title: intl.formatMessage({
+        defaultMessage: "Education",
+        description: "Title for education experience form button.",
+      }),
+      icon: <BookOpenIcon style={{ width: "1.5rem" }} />,
+    },
+    {
+      href: "/",
+      title: intl.formatMessage({
+        defaultMessage: "Award",
+        description: "Title for award experience form button.",
+      }),
+      icon: <StarIcon style={{ width: "1.5rem" }} />,
+    },
+  ];
+
+  const allExperiences = [
+    ...awardExperiences,
+    ...communityExperiences,
+    ...educationExperiences,
+    ...personalExperiences,
+    ...workExperiences,
+  ];
+
+  const sortedByDate = allExperiences.sort(compareByDate);
+
+  const allSkills: Skill[] =
+    experiences?.reduce((accumulator: Skill[], currentValue: Experience) => {
+      const skills =
+        currentValue.experienceSkills
+          ?.filter(notEmpty)
+          .map((experienceSkill) => experienceSkill.skill) || [];
+
+      return [...accumulator, ...skills];
+    }, []) || [];
+  const skillIds = allSkills.map(({ id }) => id);
+  const sortedBySkills = allSkills.filter(
```

Comment:
Should the subgroups by sorted by the date logic, too?

Reply:
Just talked to the design team and they said alphabetically.